recipes/libharu/all/conanfile.py

```diff
@@ -62,6 +68,12 @@ def _configure_cmake(self):
     def build(self):
         for patch in self.conan_data.get("patches", {}).get(self.version, []):
             tools.patch(**patch)
+        if self.options.get_safe('with_libtiff'):
+            tools.replace_in_file(os.path.join(self._source_subfolder, "src", "hpdf_image_ccitt.c"), "#define\tG3CODES", "")
+            tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"), "endif(PNG_FOUND)", '''endif(PNG_FOUND)
+
+find_package(TIFF REQUIRED)
+list(APPEND ADDITIONAL_LIBRARIES TIFF::TIFF)''')
```

Comment:
Could this be move to seperate patch file? + maybe open an issue upstream so it can get fixed?

Reply:
I think that libharu project has been inactive for several years already. There is a PR with different fix since Nov 2017 https://github.com/libharu/libharu/pull/168